app/src/main/java/org/mifos/mobile/ui/transfer_process/TransferProcessComposeFragment.kt

```diff
@@ -0,0 +1,91 @@
+package org.mifos.mobile.ui.transfer_process
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.fragment.app.viewModels
+import dagger.hilt.android.AndroidEntryPoint
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+import org.mifos.mobile.models.payload.TransferPayload
+import org.mifos.mobile.ui.enums.TransferType
+import org.mifos.mobile.ui.fragments.base.BaseFragment
+import org.mifos.mobile.utils.Constants
+import org.mifos.mobile.utils.ParcelableAndSerializableUtils.getCheckedParcelable
+import org.mifos.mobile.utils.ParcelableAndSerializableUtils.getCheckedSerializable
+
+/**
+ * Created by dilpreet on 1/7/17.
+ */
+@AndroidEntryPoint
+class TransferProcessComposeFragment : BaseFragment() {
+
+    private val viewModel: TransferProcessViewModel by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        if (activity != null) {
+            arguments?.getCheckedParcelable(
+                TransferPayload::class.java,
+                Constants.PAYLOAD
+            )?.let {
+                viewModel.setContent(it)
+            }
+            (arguments?.getCheckedSerializable(
+                TransferType::class.java,
+                Constants.TRANSFER_TYPE
+            ) as TransferType).let {
+                viewModel.setTransferType(it)
+            }
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View {
+        return ComposeView(requireContext()).apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+            setContent {
+                MifosMobileTheme {
+                    TransferProcessScreen(
+                        cancel = {
+                                closeClicked()
```

Comment:
I think replacing this with `activity?.supportFragmentManager?.popBackStack()` is a better option.